server/routes/incidentDetails/incidentDetailsValidation.ts

```diff
@@ -72,9 +72,13 @@ const errors: { [key: string]: FormError } = {
     text: 'Enter the discovery minute between 00 and 59',
   },
   DISCOVERY_FUTURE_TIME: {
-    href: '#discoveryDate[time]',
+    href: '#discoveryDate[date]',
     text: 'The incident discovery time must be in the past',
   },
+  DISCOVERY_AFTER_INCIDENT_TIME: {
+    href: '#discoveryDate[date]',
+    text: 'The incident discovery date/time must be after the indident date/time',
```

Comment:
typo should say after the incident

Reply:
This validation message needs to be run past Anne because I don't think it fits with our content standards.